### PR TITLE
Add ground-station cancel plan command.

### PR DESCRIPTION
**Pull request by danada**
Will wait on an official API release with the changes so we don't have to release the CLI that targets a snapshot.

### CODE REVIEW

**Comment by hoshir** [comment]
This PR looks marked as Draft. Can I start review?

**Comment by danada** [comment]
Yes, you can review- I marked it as a draft because it's targeting a snapshot version of the API. I don't think this should be released until the API is released again.

**Comment by hoshir** [comment]
Looks great! Thanks for updating the document as well.

**Review by hoshir — APPROVED** [review]

File: pkg/groundstation/plan/cancel_plan.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Copyright © 2018 Infostellar, Inc.
```
Comment (danada) [inline pkg/groundstation/plan/cancel_plan.go]:
```suggestion
// Copyright © 2019 Infostellar, Inc.
```

**Pull request merged**